arm-storage/2016-01-01/swagger/storage.json

```diff
@@ -463,7 +463,7 @@
       "properties": {
         "name": {
           "type": "string",
-          "description": "Gets or sets the sku name. Required for account creation, optional for update. Note that in older versions, sku name was called accountType.",
+          "description": "Gets or sets the sku name. Required for account creation, optional for update. Note that in older versions, sku name was called accountType. For Blob Storage accounts, only Standard_LRS, Standard_GRS and Standard_RAGRS are supported",
```

Comment:
Can you remove the phrase Gets or sets? Autorest generated code will automatically have the right prefix based on the property's mutability